.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Why did you do this? It needs to be an all empty file.

Reply:
Really? Interesting. I always create it with an empty "object" 🤔